releasenotes.txt

```diff
@@ -1,4 +1,23 @@
-﻿Build 5.4.0
+﻿Build 5.4.1
+=============================
+
+Release notes - NHibernate - Version 5.4.1
+
+5 issues were resolved in this release.
+
+** Bug
+
+   * #3216 Enable one-to-one optimistic lock handling in mapping
```

Comment:
Need to add 5.4.0 possible breaking change note about one-to-one update leading to parent version update and the way to return back to old behavior

Reply:
About docs: nhibernate/nhibernate.github.io#58.